opentreemap/treemap/units.py

```diff
@@ -107,6 +140,10 @@ def get_unit_name(abbrev):
     return _unit_names[abbrev]
 
 
+def get_unit_abbreviation(abbrev):
+    return _unit_abbreviations.get(abbrev, abbrev)
+
```

Comment:
get(abbrev, abbrev) as opposed to [abbrev] can delay noticing that we missed a unit. Seems to me that if we raise an exception, then if someone adds a new unit, they'll find out really quick that they missed adding its abbrev.

Are we really worried about a new unit escaping into the wild without an abbrev?